tools/wptrunner/wptrunner/testdriver-extra.js

```diff
@@ -228,6 +228,21 @@
         }
     };
 
+    const subscribe_global = async function (params) {
+        const action_result = await create_action("bidi.session.subscribe", {
+            // Subscribe to all contexts.
+            ...params
+        });
+        const subscription_id = action_result["subscription"];
+
+        return async ()=>{
+            console.log("!!@@## unsubscribing")
```

Comment:
oops. PLease remove this artifact (I know it as copied from `subscribe`, and it should be removed from there as well)